subsys/bluetooth/controller/hci_driver.c

```diff
@@ -505,6 +505,30 @@ static int ble_enable(void)
 	return 0;
 }
 
+ISR_DIRECT_DECLARE(ble_controller_radio_isr_wrapper)
+{
+	ble_controller_RADIO_IRQHandler();
+
+	ISR_DIRECT_PM();
+	return 1; /* We should check if scheduling decision should be made */
```

Comment:
Are you sure about this? The direct ISRs don't use any Zephyr RTOS functionality, so they don't affect scheduling at all. The don't unblock threads at all.